### PR TITLE
Create board connection troubleshooting for Web Editor [HC-1361]

### PR DESCRIPTION
**Pull request by Renat0Ribeir0**
- Updates [The Arduino board shows a red cross](https://support.arduino.cc/hc/en-us/articles/360018131160-The-Arduino-board-shows-a-red-cross) 
- Adds generic troubleshooting tips for board connectivity issues when using the Web Editor

### CODE REVIEW

File: content/Arduino Cloud/Web Editor/If-your-board-is-not-detected-by-Arduino-Web-Editor.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: "If your board is not detected by Arduino Web Editor"
+id: 360018131160
+---
+
+If your board is not detected by Arduino Web Editor, a red cross icon will be displayed in the board selector.
```
Comment (seaxwi) [inline content/Arduino Cloud/Web Editor/If-your-board-is-not-detected-by-Arduino-Web-Editor.md:6]:
```suggestion
If the selected board in Arduino Web Editor could not be detected, the board selector will display a red cross.
```

Slight clarification: "My board" (as in a physical board that I want to use with the Web Editor) may be detected, but if a different board is selected in the board selector (and that board is not being detected), the red cross will appear.

**Review by seaxwi — CHANGES_REQUESTED** [review]
Looks good as is, going to be a big improvement!

Left a few suggestions for you to review.

File: content/Arduino Cloud/Web Editor/If-your-board-is-not-detected-by-Arduino-Web-Editor.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+* Ensure you've connected your board with a working data USB cable.
+* Check that your device is powered. One or more more LEDs should be lit.
+* If you haven't done so already, [install Arduino Create Agent](https://create.arduino.cc/getting-started/plugin/welcome).
+* [Check if the Arduino Create Agent is installed and running](https://support.arduino.cc/hc/en-us/articles/4980687506844-Check-if-the-Arduino-Create-Agent-is-installed-and-running).
+* Open the Arduino Create Agent menu (Look for the ![Arduino Create Agent icon](img/create-agent-logo-mac.png) icon in the top-right of the menu bar for macOS and Linux or in the bottom-right of the taskbar within the system tray, for Windows) and see if `Generate and Install HTTPS Certificates` is available. If so:
+    1. Select this option
+    2. Enter password if prompted
+    3. Refresh the browser
+* Make sure to select the connected board in the board selector. Sometimes the Web Editor will show the red cross icon and the name of a board that has been previously connected to the Web Editor:
```
[the remark this answers is not in the document]
Comment (Renat0Ribeir0) [reply content/Arduino Cloud/Web Editor/If-your-board-is-not-detected-by-Arduino-Web-Editor.md]: https://github.com/arduino/help-center-content/pull/271/commits/24cdb5f2eb71820f1454f69bae46a8209c2c680c

**Review by seaxwi — APPROVED** [review]
LGTM 👍

**Pull request merged**